account_analytic_parent/models/account_analytic_account.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Luxim d.o.o.
+# Copyright 2017 Matmoz d.o.o.
+# Copyright 2017 Deneroteam.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = 'account.analytic.account'
+
+    parent_id = fields.Many2one(
```

Comment:
It's not a question of usability. It's a question of availability. You don't have available on each of the places where analytic accounts are displayed (sale, purchase, invoice, moves...) the domain for filtering out "view" accounts unless you depend on all of them, so this other field makes the trick, although it mixes 2 concepts, but in some way related, as the goal is the same: to not allow to select that accounts. Anyway, there's a possible way of differentiating them: deactivated account with children, it's a view account. If not, it's a deactivated account.

I put an example of the glue modules you will need it:
```
from odoo import fields, models

class PurchaseOrderLine(models.Model):
    _inherit = 'purchase.order.line'

    analytic_account_id = fields.Many2one(domain="[('type', '!=', 'view')]")
```